_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -109,9 +111,9 @@ Lets take a look at the pinger HTML, and see how it works.
 
```

Comment:
```suggestion
```

There's an extra blank line after the `</html>`. It should be trimmed.

Reply:
Thanks! Pushed.